sky/setup_files/dependencies.py

```diff
@@ -9,7 +9,7 @@
 from typing import Dict, List
 
 install_requires = [
-    'wheel',
+    'wheel<0.46.0',
```

Comment:
Add a quick comment with reference to the issue?
```suggestion
    # https://github.com/skypilot-org/skypilot/issues/5153
    'wheel<0.46.0',
```